src/data/fin.lean

```diff
@@ -959,7 +972,7 @@ end fin
 -- as a duplicate of `function.embedding.subtype`.
 /-- Embedding of `fin n` into `ℕ`. -/
 def function.embedding.fin (n : ℕ) : fin n ↪ ℕ :=
-⟨coe, fin.val_injective⟩
+⟨coe, subtype.coe_injective⟩
 
 -- Once lean#359 is fixed (making `fin n` a subtype), this can go away
 -- as a duplicate of `function.embedding.coe_subtype`.
```

Comment:
These two definitions (which I never ended up using) can, as the comments on them note, be removed now that `fin` is a subtype.

Reply:
Removed these defs.